contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -6,8 +6,8 @@ check() {
 	[ "${1}" = "-d" ] && return 0
 
 	# Verify the zfs tool chain
-	for tool in "@sbindir@/zgenhostid" "@sbindir@/zpool" "@sbindir@/zfs" "@mounthelperdir@/mount.zfs" ; do
-		test -x "$tool" || return 1
+	for tool in "zgenhostid" "zpool" "zfs" "mount.zfs"; do
+		command -v "${tool}" >/dev/null 2>&1 || return 1
```

Comment:
`2>&1` is erroneous here (actually, on L45, too, but it was there in the original?), you *want* to see any errors. Besides that, looks great, no further comments.

Reply:
Done! Changed it in both places. I am not a career programmer, so thank you to both you and @ahesford for walking me through this process.